path.c

```diff
@@ -1232,6 +1232,11 @@ int change_directory(
 
     assert(!logical || !ensure_pwd);
 
+    if ( newpwd[0] == L'\0' ) {
+	xerror(0, Ngt("empty directory"));
```

Comment:
Thanks for adding the new message, but "empty directory" might sound like the directory has no files in it, rather than the directory name being zero length. How about "empty directory name"?

```suggestion
	xerror(0, Ngt("empty directory name"));
```

Reply:
Yes, you're right. "empty directory name" sounds better